packages/graph/hash_graph/lib/graph/src/api/web/data_type.rs

```diff
@@ -0,0 +1,145 @@
+//! Web routes for Data Types CRU operations.
```

Comment:
```suggestion
//! Web routes for CRU operations on Data Types .
```

or 

```suggestion
//! Web routes for Data Types' CRU operations.
```

Reply:
Addressed in https://github.com/hashintel/hash/pull/750/commits/be9e2c7dbb0ef29141a1a190a3fbd3bdc99294c0